packages/esm-patient-registration-app/src/config-schemas/openmrs-esm-patient-registration-schema.ts

```diff
@@ -27,8 +27,8 @@ export const esmPatientRegistrationSchema = {
       },
     },
     _default: {
-      demographics: { name: 'Basic Info', fields: ['name', 'gender', 'dob', 'id'] },
-      contact: { name: 'Contact Details', fields: ['address', 'phone', 'email'] },
+      demographics: { name: 'Basic Info', fields: ['name', 'gender & dob', 'id'] },
+      contact: { name: 'Contact Details', fields: ['address', 'phone & email'] },
```

Comment:
In general, I think the configuration option(s) here seem to make almost no sense - at least when looking at the very static and fixed design.

We need to ping this back.

What makes sense here regarding configurability? How is this reflected in the design? We need this to be answered.

Reply:
These are being used to show the fields in the form. The form is dynamic in relation to the fields on the basis of these configuration. You might know about this before, so maybe this is not what you wanted as an answer. 
Who should I contact to clarify this properly?

Reply:
> These are being used to show the fields in the form. The form is dynamic in relation to the fields on the basis of these configuration.

Tell me something I don't know.

> Who should I contact to clarify this properly?

Every stakeholder. This should be a general discussion point about configuration options. Right now, some options are either

1. too far fetched and will never be used
2. too complicated and will never be used
3. too limited and will result in frustration
4. conflicting with the design choices (this is a good example)

The problem is - and I've mentioned that several times already - that the design is done as a monolith. It does not care about

- the micro frontend nature of the application (i.e., everything can grow; even without us doing something)
- the configuration possibilities
- the potential design overrides (ex. Brandon's custom navbar)

But these things need to be on the table. They need to be very present. They are at the heart of the application and dismissing them from the design is an issue.

cc @jonathandick